aip-console-jenkins/src/main/java/io/jenkins/plugins/aipconsole/AddVersionBuilder.java

```diff
@@ -462,11 +463,14 @@ protected void performClient(@Nonnull Run<?, ?> run, @Nonnull FilePath workspace
             } else {
                 log.println(AddVersionBuilder_AddVersion_info_startAddVersionJob(variableAppName));
             }
+
+            boolean expandedSecurityDataflow = isSecurityDataflow() || Boolean.valueOf(run.getEnvironment(listener).get("SECURITY_DATAFLOW"));
```

Comment:
maybe name this variable isSecurityDataflowEnabled ? or, since the DeliverBuilder class defines the same variable and computes its value the same way, create a method similar to "isEnableDataSafety()" that returns `isSecurityDataflow() || Boolean.valueOf(run.getEnvironment(listener).get("SECURITY_DATAFLOW"))` that would be called here and also at line #507 of DeliverBuilder.java

Reply:
ok